Create introduction.md

slushpupie opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

slushpupie committed:
Create introduction.md

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+# Introduction
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

slushpupie replied:
Indeed!

slushpupie committed:
Update introduction.md

slushpupie committed:
[from now] 2016/11/13 20:37:27

diff --git a/introduction.md b/introduction.md
index 023ea01..07c744d 100644
--- a/introduction.md
+++ b/introduction.md
@@ -3,3 +3,8 @@
 This is an introduction

 Some mention about the Ruby path
+
+ * Java
+ * Git
+ * Linux
+ * Grue Slaying

slushpupie committed:
[from now] 2016/11/13 20:39:30

diff --git a/introduction.md b/introduction.md
index 07c744d..9b66bc6 100644
--- a/introduction.md
+++ b/introduction.md
@@ -8,3 +8,4 @@ Some mention about the Ruby path
  * Git
  * Linux
  * Grue Slaying
+ * Killing history using git rebase

codeschool-kiddo committed:
Added new rule about coffee

slushpupie committed:
[from now] 2016/11/13 20:41:56

diff --git a/dojo_rules.md b/dojo_rules.md
index 3569c40..6a1df12 100644
--- a/dojo_rules.md
+++ b/dojo_rules.md
@@ -6,4 +6,4 @@ Dojo Rules
 * Never bring live blades on the tatami
 * Respect your opponents
 * Most Sensei's should be respected
-* no coffe in the Dojo
\ No newline at end of file
+* no coffee in the dojo

slushpupie committed:
Merge pull request #2 from slushpupie/new_rules

Too many people are spilling coffee

slushpupie committed:
Merge branch 'deadly_skills'

slushpupie committed:
[from now] 2016/11/13 20:57:40

diff --git a/kill_list.md b/kill_list.md
index 520bb8b..4c9a91f 100644
--- a/kill_list.md
+++ b/kill_list.md
@@ -1,6 +1,7 @@
 Kill List
 =========
-* O-Ren Ishii
-* Vernita Green
-* Elle Driver
-* BILL
+* Unused code
+* Functions with side effects
+* Unformatted code
+* Repeated code blocks
+* Bad variable names

slushpupie committed:
Merge branch 'release_branch_1.0'

slushpupie committed:
Stuff

Fixes #3

slushpupie committed:
Merge pull request #4 from slushpupie/kill_list

Stuff

slushpupie committed:
Update README.md
